packages/design-tokens/src/getters.ts

```diff
@@ -20,16 +13,24 @@ import {
   TypographySize,
   ZIndexGroup,
   StyledComponentsInterpolation,
+  Theme,
 } from './types'
 
-export function getBreakpoint(size: BreakpointSize): Breakpoint | undefined {
+function getTheme(theme?: Theme): Theme {
+  return theme?.coloursTokens ? theme : defaultTheme
```

Comment:
It could be good to use `color` spelling although I appreciate we use `colour` in other places.

Reply:
I agree and the selector uses `color` but I have a feeling I couldn't easily change @hxltrhuxze original spelling in the tokens because of the css-framework. I'll investigate.